### PR TITLE
staticpod: switch controllers to use context instead of stopCh

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/operator/staticpod/controllers.go [hunk 1]
```diff
@@ -157,11 +165,13 @@ func (b *staticPodOperatorControllerBuilder) ToControllers() (*staticPodOperator
 			configMapClient,
 			secretClient,
 			eventRecorder,
-		)
+		))
+	} else {
+		errs = append(errs, fmt.Errorf("missing revisionController; cannot proceed"))
```
Comment (sttts) [inline pkg/operator/staticpod/controllers.go]: why no hard return err?

Comment (mfojtik) [reply pkg/operator/staticpod/controllers.go]: the original idea was to collect all errors I think, that is pre-existing.

Comment (sttts) [reply pkg/operator/staticpod/controllers.go]: ic. The controller names are not necessary anymore.

**Comment by sttts** [comment]
/lgtm
/approve

Happy bumping!

**Pull request merged**